config/core/test-configs.yaml

```diff
@@ -14,6 +14,12 @@ file_system_types:
       x86:     [{arch: i386}, {arch: x86_64}]
       mipsel:  [{arch: mips}]
 
+  cip:
+    url: 'https://storage.staging.kernelci.org/images/rootfs/cip/20210910/'
```

Comment:
Note: We will need to upload the images on https://storage.kerneci.org before deploying this in production.

Reply:
yes, I need a production token for upload to production storage.